src/defines.h

```diff
@@ -9,7 +9,7 @@
 #include <math.h>
 
 // Sample rate before decimation
-#define SAMPLE_RATE 1488375
+#define SAMPLE_RATE NRSC5_SAMPLE_RATE_CU8
```

Comment:
Do you suppose we should get rid of `SAMPLE_RATE` and just use `NRSC5_SAMPLE_RATE_CU8` instead?